packages/react-examples/.storybook/preview-loader.js

```diff
@@ -9,7 +8,7 @@ import * as path from 'path';
  * Also if this is the storybook for `@fluentui/react` or `@fluentui/react-components`, replace the
  * REACT_DEPS token with the actual list of dependencies which have their own examples but should be
  * included in the suite package storybook.
- * @this {*} loader API https://webpack.js.org/api/loaders/
+ * @this {{addDependency: (file: string)=>void;callback: ((err: Error | null)=>void)}} loader context - https://webpack.js.org/api/loaders/#the-loader-context
```

Comment:
wasn't able to find proper Context typings in webpack source

Reply:
Yeah they removed it in version 5. 😢 There's an open issue: https://github.com/webpack/webpack/issues/11630

Reply:
wow 6 months and not resolved yet. The approach regarding types from webpack team is not an ideal one...